src/raster/r.forestfrag/r.forestfrag.html

```diff
@@ -109,14 +114,3 @@ <h2>AUTHORS</h2>
 <li>Paulo van Breugel (Python version, user-defined moving window size)</li>
 <li>Vaclav Petras (major code clean up)</li>
 </ul>
```

Comment:
All other authors are not listed with bullets. Shall we change this?

Reply:
Yes, will remove it.